### PR TITLE
Create getIntervals and cardsToNotes functions

### PR DESCRIPTION
**Pull request by techdavid**
getIntervals lists the most recent interval, or all intervals, of the given cards, and cardsToNotes returns an unordered array of note IDs for the given cards.

### CODE REVIEW

File: AnkiConnect.py [hunk 1]
```diff
@@ -409,6 +409,16 @@ def isSuspended(self, card):
             return False
 
 
+    def getIntervals(self, cards, complete=False):
+        intervals = []
+        for card in cards:
+            interval = self.window().col.db.list("select ivl from revlog where cid = ?", card)
```
Comment (FooSoft) [inline AnkiConnect.py]: Single quote strings please

**Comment by FooSoft** [comment]
Looks good, fix the quoted string and I'll merge it in 👍

**Comment by techdavid** [comment]
Fixed (1a2c559 and e02f8b2 amended) - had been working on some bugfixes Anki a couple of days ago and got used to double quotes. I'll try to remember for next time 😉

**Comment by FooSoft** [comment]
Thanks for the code!

**Pull request merged**